.github/workflows/e2e-test.yaml

```diff
@@ -46,23 +46,30 @@ jobs:
         run: |
           curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
           source /home/runner/.profile
-          tiup update --nightly
-          tiup playground nightly --tiflash=0 &
+          tiup install tidb:v4.0.5 tikv pd prometheus
+          tiup install playground
+          tiup update --all
+          tiup update --self
+          source /home/runner/.profile
+          tiup playground --tiflash=0 &
       - name: Build UI
         run: |
           make ui
         env:
           NO_MINIMIZE: true
           CI: true
           REACT_APP_MIXPANEL_TOKEN: ""
+      # - name: Wait TiUP Playground
+      #   run: |
+      #     chmod u+x scripts/wait_tiup_playground.sh
+      #     scripts/wait_tiup_playground.sh 15 20
```

Comment:
Will uncomment it when tiup release its new version which fixed the bug that `tiup playground display` command doesn't work.